datacite/rest_client.py

```diff
@@ -25,49 +25,35 @@
 class DataCiteRESTClient(object):
     """DataCite REST API client wrapper."""
 
-    def __init__(self, username=None, password=None, url=None, prefix=None,
-                 test_mode=False, api_ver="2", timeout=None):
+    def __init__(self, username, password, url, prefix, timeout=None):
```

Comment:
Per discussion, update to `def __init__(self, username, password, prefix, test_mode=True, url=None, timeout=None):